README.md

```diff
@@ -1,11 +1,119 @@
+
+![Furhter Context on one of the improvements being tackled](docs/img/SpecificProjectStructureExpand.png)
+
+#### Correct Document's Grammar
+
+`Papyrus: Correct document's grammar`
+
+Opens a [Split Screen Editor](#split-screen-editor) with a version of the document with it's grammar corrected.
+
+![Correct Grammer Command Example](docs/img/CorrectGrammar.example.png)
+
+#### Chat with a Custom Persona
+
+`Papyrus: Chat with a custom Persona`
+
+With this command you can create a text chat with one of the prederfined personality or with a custom one. The personality has access to the document currently being edited.
+
+Available Personas:
+- Developer
+- Influencer
+- Project Manager
+- Xavier
+- Money Guy
+- Cook
+- Tech Blogger
+
+![Selecting Text Blogger Persona](docs/img/SelectTextBloggerPersona.example.png)
+
+![Define Custom Persona](docs/img/CustomPersona.example.png)
+
+![Talking With Text Blogger Persona](docs/img/ChattingWithTextBloggerPersona.example.png)
+
+#### Import Template to Document
+
+`Papyrus: Import template to document`
+
+Pick a template you would like to apply and the plugin will open a slit screen with the current opened document formatted to fit the selected template.
+
+#### Extract Template from Document
+
+`Papyrus: Extract template from document`
+
+This command analyses the provided document and creates a template which can be then used as a starting point for future similar documents.
+
+You can define the name of the file in which it will be stored. That filed is then created under the folder `/templates`
+
+![alt text](docs/img/ExtractTemplatePrompt_example.png)
+
+#### Create Actionable Tasks from Document
+
+`Papyrus: Create actionable tasks from document`
+
+This command analysis the provided document and creates a list of actionables. Each listed actionable is presented with the following properties:
+- Reasoning - the reasoning behind the proposition of the task
+- Category - the cotegory of the presented actionable
+- Subtasks - Specifications of the presented actionable
+- Dependencies - (if applicable) Skills needed to perform the task
+- Priority, Cost - Perceived priority and cost to perform the task, so the user can gadge whether to do it or not
+
+![Example of the Create Actionables command](docs/img/ActionableExample.png)
+
+Then those tasks are added to the document as checkbox at the end of it.
+
+#### Translate Document to Another Language
+
+`Papyrus: Translate document to another language`
+
+It translates the document into English. Opens a [Split Screen Editor](#split-screen-editor) with the translated version.
+
+![Note Translated from Portuguese to English](docs/img/TranslateDocumentToAnotherLanguage.example.png)
+
+## Contributing
+
+Contributions via bug reports, but fixes, documentation, and general improvements are always welcome. For more major feature work, make an issue about the future idea / reach out to the team so we can sudge feasibility and how best to implement it.
```

Comment:
"For more major feature work" Não bastava dizer "For new features"? Acho que fica mais simples e mais claro